### PR TITLE
Apex Assignment1 commit

### PR DESCRIPTION
**Pull request by kumarnitabh**

### CODE REVIEW

File: classes/Assignment1.cls [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class Assignment1 {
```
Comment (shubhamAgarwal242) [inline classes/Assignment1.cls:1]: @kumarnitabh, Please update the class name.
Class name should be proper with a meaning

File: classes/Assignment1.cls [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    public static void Assignment1() {
```
Comment (shubhamAgarwal242) [inline classes/Assignment1.cls:2]: @kumarnitabh, Please update the method name as well

File: classes/Assignment1Part2.cls [hunk 3]
```diff
@@ -0,0 +1,45 @@
+public  class Assignment1Part2 {
+    public static void Assignment1Part2() {
+
+        // Set1 of students 
+        
+        Set<String> StudentSet1 = new Set<String>();
+        
+        // Adding values in Set1 
+
```
Comment (shubhamAgarwal242) [inline classes/Assignment1Part2.cls:9]: @kumarnitabh, this type of line space is not needed

File: classes/Assignment1Part3.cls [hunk 4]
```diff
@@ -0,0 +1,19 @@
+public  class Assignment1Part3 {
+    public static void Assignment1Part3() {
+
+        List<Integer> NumbersList =new List<Integer>();
+            
+        List<Integer> RandomNumbers=new List<Integer>{3,43,12,76,11,32,91,62,72,37};
```
Comment (shubhamAgarwal242) [inline classes/Assignment1Part3.cls:6]: @kumarnitabh, variable name should be followed up camel case

**Review by shubhamAgarwal242 — CHANGES_REQUESTED** [review]

File: classes/EmployeeDepartmentMap.cls [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+ * Description : Mapping a list
+ * 
+ * Created By : Kumar Nitabh
+ * 
+ * Created on : 26/11/2024
+ * 
+ */
+
+public class EmployeeDepartmentMap {
+    public static void employeeData() {
```
Comment (shubhamAgarwal242) [inline classes/EmployeeDepartmentMap.cls:11]: @kumarnitabh, please add the method comments as well

File: classes/EmployeeDepartmentMap.cls [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+        Map<String, List<String>> dataOfEmployee =new Map<String, List<String>>();
```
Comment (shubhamAgarwal242) [inline classes/EmployeeDepartmentMap.cls:13]: @kumarnitabh, Use proper variable name like : mapOfDepartmentToEmployees

File: classes/OperationsOnList.cls [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+ * Created By : Kumar Nitabh
+ * 
+ * Created on : 26/11/2024
+ * 
+ */
+
+public  class OperationsOnList {
+    public static void sortedList() {
+
+        List<Integer> numbersList =new List<Integer>();
```
Comment (shubhamAgarwal242) [inline classes/OperationsOnList.cls:13]: @kumarnitabh, As discussed, variable name should be like: listOfNumbers

File: classes/OperationsOnList.cls [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+ * Description : Created a list add random numbers then sort it 
+ * 
+ * Created By : Kumar Nitabh
+ * 
+ * Created on : 26/11/2024
+ * 
+ */
+
+public  class OperationsOnList {
+    public static void sortedList() {
```
Comment (shubhamAgarwal242) [inline classes/OperationsOnList.cls:11]: @kumarnitabh, Add method comments

File: classes/OperationsOnSet.cls [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+ * Description : Take 2 sets and perform Union, Intersection and Difference sets.
+ * 
+ * Created By : Kumar Nitabh
+ * 
+ * Created on : 26/11/2024
+ * 
+ */
+
+public  class OperationsOnSet {
+    public static void setDifferentOperation () {
```
Comment (shubhamAgarwal242) [inline classes/OperationsOnSet.cls:11]: @kumarnitabh, Please add the method comments

File: classes/OperationsOnSet.cls [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+        // Set1 of students 
+        Set<String> studentSet1 = new Set<String>();
```
Comment (shubhamAgarwal242) [inline classes/OperationsOnSet.cls:14]: @kumarnitabh, variable name should be setOfStudents1

**Review by shubhamAgarwal242 — CHANGES_REQUESTED** [review]

File: classes/EmployeeDepartmentMap.cls (hunk 5 above)
Comment (kumarnitabh) [reply classes/EmployeeDepartmentMap.cls]: Resolved

**Review by shubhamAgarwal242 — APPROVED** [review]

**Pull request merged**